open-metadata-implementation/access-services/security-officer/security-officer-api/src/main/java/org/odpi/openmetadata/accessservices/securityofficer/api/events/SecurityOfficerEventType.java

```diff
@@ -9,7 +9,8 @@
 
 public enum SecurityOfficerEventType implements Serializable {
 
-    NEW_SECURITY_ASSIGNMENT(0, "NewSecurityAssignment", "A security tag assigment should be processes."),
+    NEW_SECURITY_ASSIGNMENT(0, "NewSecurityAssignment", "A Security Tags assigment should be processes."),
```

Comment:
Should this say 'should be processed' ?

Reply:
yes, thank you! I've done the update in the last commit.